src/resources/assets/scss/customs/_list.scss

```diff
@@ -0,0 +1,335 @@
+// DataTables
+#crudTable_wrapper {
+  display: block;
+
+  .dataTables_processing {
+    background: rgba(255, 255, 255, 0.9);
+    top: 26px;
+    left: 115px;
+    height: calc(100% - 6px);
+    width: calc(100% - 30px);
+    z-index: 999;
+    border: none;
+    box-shadow: none;
+
+    &>img {
+      top: calc(50% - 16px);
+      left: calc(50% - 16px);
+      position: absolute;
+    }
+  }
+
+  .dataTables_length select {
+    min-width: 3.6rem;
+    margin-right: 0.4rem;
+  }
+
+  #crudTable {
+    margin-top: 0 !important;
+
+    &.dtr-inline.collapsed>tbody>tr>.dtr-control:before {
+      background-color: transparent;
+      color: #636161;
+      font-family: 'Line Awesome Free';
+      font-weight: 900;
+      width: 16px;
+      content: "\f142";
+      font-size: 21px;
+      box-shadow: none;
+      border: none;
+      display: inline;
+      position: relative;
+      top: 0;
+      left: 0;
+      margin: 0 0 0 -.25rem;
+    }
+
+    &.dtr-inline.collapsed>tbody>tr>.dtr-control:before {
+      top: 0;
+    }
+
+    td, th {
+      outline: none;
+      vertical-align: middle;
+    }
+
+    thead > tr {
+      &>th:not(.sorting_disabled),
+      &>td {
+        padding-right: 30px;
+      }
+    }
+
+    .sorting:before,
+    .sorting_asc:before,
+    .sorting_desc:before,
+    .sorting_asc_disabled:before,
+    .sorting_desc_disabled:before {
+      right: 0.4em;
+      top: 1em;
+      content: "\f0d8";
+      font: normal normal normal 14px/1 "Line Awesome Free";
+      font-weight: 900;
+    }
+
+    .sorting:after,
+    .sorting_asc:after,
+    .sorting_desc:after,
+    .sorting_asc_disabled:after,
+    .sorting_desc_disabled:after {
+      right: 0.4em;
+      content: "\f0d7";
+      font: normal normal normal 14px/1 "Line Awesome Free";
+      font-weight: 900;
+    }
+
+    tbody>tr:first-of-type>td:first-of-type,
+    tbody>tr:not(.shown)+tr>td:first-of-type,
+    thead>tr>th:first-of-type,
+    tfoot>tr>th:first-of-type {
+      padding: 0;
+      min-height: 1rem;
+    }
+
+    &.has-hidden-columns .details-control {
+      margin-left: .4rem;
+    }
+
+    &.has-hidden-columns {
+      thead tr th:first-child,
+      tfoot tr th:first-child {
+        padding-left: 1.35rem!important;
+      }
+    }
+
+    .details-control {
+      display: flex;
+    }
+
+    .crud_bulk_actions_checkbox {
+      display: flex;
+      margin: 0 0 0 .625rem;
+    }
+
+    .crud_bulk_actions_checkbox input {
+      width: 1rem;
+      height: 1rem;
+    }
+
+    &:not(.has-hidden-columns) {
+      tr td:first-child {
+        padding-top: 1.25rem!important;
+      }
+    }
+    tr th:first-child,
+    tr td:first-child {
+      display: flex;
+      align-items: center;
+      padding: 1rem 0 1rem 0.6rem!important;
+    }
+
+    &.has-details-row {
+      thead tr th:first-child,
+      tfoot tr th:first-child {
+        padding-left: 2.1rem!important;
+      }
+    }
+

```

Comment:
My experience in Monsters, with both details row and the three dots, wasn't the same as yours. Here's what I see on mobile in Monsters CRUD:

![2021-12-27 09 05 13](https://user-images.githubusercontent.com/1032474/147444705-c193c524-5f0e-4c93-8ab6-31891004c248.gif)


Some pages have just the three dots, but extra space.
Other pages have both the three dots and details row icon, but the text doesn't show anymore (see my other comment here about column `limit`).

Reply:
This one is fixed, by making the first column having priority in responsive.